conbench/templates/compare-entity.html

```diff
@@ -84,9 +83,7 @@
           </li>
           <li class="list-group-item" style="overflow-y: auto;">
             <b>case permutation</b>
-            <div align="right" style="display:inline-block; float: right;">
-              <a href="{{ url_for('app.benchmark-result', benchmark_result_id=baseline.id) }}">{{ baseline.display_case_perm }}</a>
-            </div>
+            <div align="right" style="display:inline-block; float: right;">{{ baseline.display_case_perm }}</div>
```

Comment:
Much better!

Reply:
Thanks! Yeah the main point of showing the result ID was to have _that_ link to the result, and not have the case perm link to the result.